napari/_vispy/layers/image.py

```diff
@@ -160,7 +161,7 @@ def _on_data_change(self) -> None:
         # Check if ndisplay has changed current node type needs updating
         if (ndisplay == 3 and not isinstance(node, VolumeNode)) or (
             ndisplay == 2
-            and not isinstance(node, ImageNode)
+            and not isinstance(node, ImageVisual)
```

Comment:
This is really important change in context of performance. 
For current main we refresh node on every change of layer, as `LabelNode` do not inherits from `ImageNode`

Reply:
What does "refresh node" mean, and what is the alternative?

Reply:
I do not understand the question.

Reply:
You said "For current main we refresh node on every change of layer, as LabelNode do not inherits from ImageNode". What does "refresh node" mean?